src/pages/Payment/Payment.jsx

```diff
@@ -4,18 +4,28 @@ import ShippingAddress from './component/ShippingAddress';
 import './Payment.scss';
 
 const Payment = () => {
-  const [paymentPaymentProductList, setPaymentProductList] = useState([]);
-
+  const [PaymentProductList, setPaymentProductList] = useState([]);
+  const [foodList, setFoodList] = useState([]);
+  const possessionPoint = PaymentProductList[0] && PaymentProductList[0].point;
+  const foodPriceList = foodList.map(item => item.price * item.quantity);
+  const foodPriceSum = foodPriceList.reduce(
+    (accumulator, currentValue) => accumulator + currentValue,
+    0
+  );
```

Comment:
```suggestion
const totalPrice = foodList.reduce((acc,cur)=>acc+cur.price*cur.quantity,0)
```
map() 없이 reduce로 한번에 처리할 수 있겠습니다.

Reply:
피드백 반영했습니다.